### DISCUSSION TITLE
Is brew update or upgrade suppose to update cask apps as well?

### DISCUSSION THREAD

**Discussion by Yaarik** [Homebrew/discussions#1832] [Casks]
Scenario: 
1. I installed Lastpass using the command here https://formulae.brew.sh/cask/lastpass
2. Installed version 4.75.x
3. Inside the app I am getting an "new version available" 4.76.x
4. Looking at the page https://formulae.brew.sh/cask/lastpass I can see that the "Current version: 4.76.0,1181"
5. Running `brew update` or `brew upgrade` does not update the app to version 4.76.x

Am I missing something here?

**Comment by SMillerDev** [comment]
The lastpass app can auto-update, so brew won't interfere with that by default. You could pass `--greedy` to `brew upgrade` if you absolutely want brew to update the app though.

**Reply by Yaarik** [reply]
Thanks. I want to be greedy :D 

Appreciate the feedback @SMillerDev !
